AmplifyPlugins/DataStore/Tests/DataStoreHostApp/DataStoreStressTests/README.md

```diff
@@ -0,0 +1,250 @@
+## DataStore Stress Tests
+
+The following steps demonstrate how to set up DataStore with a conflict resolution enabled API through amplify CLI, with API key authentication mode. 
+
+
+### Set-up
+
+1. `amplify init` and choose `iOS` for type of app you are building
+
+2. `amplify add api`
+
+```perl
+? Please select from one of the below mentioned services: `GraphQL`
+? Provide API name: `apiName`
+? Choose the default authorization type for the API `API key`
+? Enter a description for the API key:
+? After how many days from now the API key should expire (1-365): `365`
+? Do you want to configure advanced settings for the GraphQL API `Yes, I want to make some additional changes.`
+? Configure additional auth types? `No`
+? Configure conflict detection? `Yes`
+? Select the default resolution strategy `Auto Merge`
+? Choose a schema template: `Blank Schema`
+? Do you want to edit the schema now? Y
+```
+When asked to provide the schema, create the `schema.graphql` file
+```
+enum PostStatus {
+    PRIVATE
+    DRAFT
+    PUBLISHED
+}
+
+type Post @model {
+    id: ID!
+    title: String!
+    content: String!
+    createdAt: AWSDateTime!
+    updatedAt: AWSDateTime
+    draft: Boolean
+    rating: Float
+    status: PostStatus
+    comments: [Comment] @connection(name: "PostComment")
+}
+
+type Comment @model {
+    id: ID!
+    content: String!
+    createdAt: AWSDateTime!
+    post: Post @connection(name: "PostComment")
+}
+
+## These are examples from https://docs.amplify.aws/cli/graphql-transformer/connection
+
+# 1 - Project has a single optional Team
+type Project1 @model {
```

Comment:
Since these models are in the schema but not used in the test, is there a future plan to add different types of stress tests against them? Can you comment a bit on what future tests and their correlation to the remaining model graphs? Having them here now will make it easier to build the new tests in the future without having to run `amplify push` with the updated schema. We can just write new tests and reuse the same backend. If there are no future plans, i would prefer to remove unused models